From a68147248180e7d0c3963eb35adf386b5f938164 Mon Sep 17 00:00:00 2001
From: Lulu Cheng <chenglulu@loongson.cn>
Date: Wed, 28 Sep 2022 16:35:06 +0800
Subject: [PATCH 098/133] LoongArch: Fixed a typo in the comment information of
 the function loongarch_asan_shadow_offset.

gcc/ChangeLog:

	* config/loongarch/loongarch.cc (loongarch_asan_shadow_offset):
	Fixed typo in "asan_mapping.h".

Signed-off-by: Peng Fan <fanpeng@loongson.cn>
---
 gcc/config/loongarch/loongarch.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/loongarch/loongarch.cc b/gcc/config/loongarch/loongarch.cc
index da52eefef8d..c6482c21de1 100644
--- a/src/gcc/config/loongarch/loongarch.cc
+++ b/src/gcc/config/loongarch/loongarch.cc
@@ -6583,7 +6583,7 @@ static unsigned HOST_WIDE_INT
 loongarch_asan_shadow_offset (void)
 {
   /* We only have libsanitizer support for LOONGARCH64 at present.
-     This value is taken from the file libsanitizer/asan/asan_mappint.h.  */
+     This value is taken from the file libsanitizer/asan/asan_mapping.h.  */
   return TARGET_64BIT ? (HOST_WIDE_INT_1 << 46) : 0;
 }
 
-- 
2.43.4

